File: Python/Program/Pendingin.py
from Komponen import Komponen

class Pendingin(Komponen):
    def __init__(self, merk, nama, tipe, kecepatanRPM):
        super().__init__(merk, nama)
        self.tipe = tipe
        self.kecepatanRPM = kecepatanRPM
        
    def set_tipe(self, tipe):
        self._tipe = tipe
        
    def get_tipe(self):
        return self._tipe
    
    def set_kecepatanRPM(self, kecepatanRPM):
        self._kecepatanRPM = kecepatanRPM
        
    def get_kecepatanRPM(self):
        return self._kecepatanRPM

File: Java/Program/Pendingin.java
public class Pendingin extends Komponen {
    private String tipe;
    private int kecepatanRPM;

    public Pendingin(String merk, String nama, String tipe, int kecepatanRPM) {
        super(merk, nama);
        this.tipe = tipe;
        this.kecepatanRPM = kecepatanRPM;
    }

    public void setTipe(String tipe) {
        this.tipe = tipe;
    }

    public String getTipe() {
        return tipe;
    }

    public void setKecepatanRPM(int kecepatanRPM) {
        this.kecepatanRPM = kecepatanRPM;
    }

    public int getKecepatanRPM() {
        return kecepatanRPM;
    }
}